bots/image-refresh

```diff
@@ -138,6 +139,10 @@ def run(image, verbose=False, **kwargs):
         # we want to validate all image refreshes with external projects
         task.label(pull, ["test-external"])
 
+        # HACK: sometimes statuses get eaten if posting statuses right after
+        # creating the pull request
+        time.sleep(5)
```

Comment:
Could this become a retry loop rather? I. e. trying to api.get the status afterwards?

Reply:
of course